db/migrate/20181023152814_add_first_name_and_last_name_to_users.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddFirstNameAndLastNameToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :first_name, :string
```

Comment:
The DB and the model should be consistent, if you have a validation in the model you should have a validation in the DB.